src/common/gui/theme.ts

```diff
@@ -95,30 +77,18 @@ export const themeOptions = (isCalendarApp: boolean) =>
 		},
 	] as const
 
-export function getContentButtonIconBackground(): string {
-	return theme.content_button_icon_bg || theme.content_button // fallback for the new color content_button_icon_bg
-}
-
-export function getNavButtonIconBackground(): string {
-	return theme.navigation_button_icon_bg || theme.navigation_button // fallback for the new color content_button_icon_bg
-}
-
 export function getElevatedBackground(): string {
-	return theme.elevated_bg || theme.content_bg
+	return isColorLight(theme.surface) ? theme.surface : theme.surface_container
```

Comment:
I was checking and it seems like m3 just doesn't use specific color for elevation anymore? it's kind of on case-by-case basis which container you pick So I wonder if we should just pick one. We can still  do it later though